src/client-entry.ts

```diff
@@ -38,6 +38,14 @@ factory.setEntityRepository( new EntityRepository(
 	( window as MwWindow ).mw.hook( Hooks.entityLoaded ),
 ) );
 
+factory.setEntityEditabilityResolver( {
+	isEditable() {
+		return Promise.resolve(
+			( window as MwWindow ).mw.config.get( 'wgIsProbablyEditable' ),
```

Comment:
`wgIsProbablyEditable` is missing in `src/mock-data/MwConfig.ts`.

Reply:
done